File: src/main/java/com/example/domain/review/dto/response/ReviewDeleteResponse.java
package com.example.domain.review.dto.response;

import com.example.domain.review.entity.Review;
import com.example.domain.review.entity.ReviewStatus;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class ReviewDeleteResponse {

    private Long reviewId;
    private ReviewStatus status;

    public static ReviewDeleteResponse from(Review review) {
        return new ReviewDeleteResponse(
                review.getId(),
                review.getStatus()
        );
    }
}

File: src/main/java/com/example/domain/review/dto/response/ReviewDeleteResponse.kt
package com.example.domain.review.dto.response

import com.example.domain.review.entity.Review
import com.example.domain.review.entity.ReviewStatus

data class ReviewDeleteResponse(
    val reviewId: Long?,
    val status: ReviewStatus?
) {
    companion object {
        @JvmStatic
        fun from(review: Review): ReviewDeleteResponse {
            return ReviewDeleteResponse(
                reviewId = review.id,
                status = review.status
            )
        }
    }
}